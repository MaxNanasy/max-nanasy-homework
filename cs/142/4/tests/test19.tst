type t1 = array 10 of int;
short proc(short a) { return a; }
main()
{
  var a : t1;
  const c = 11;
  a[1] = ::proc(c);
  if (a[1] == c)  {
     ERROR
  }
}
