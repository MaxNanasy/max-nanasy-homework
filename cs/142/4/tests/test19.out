Error: SEMANTIC:  UNDEFINED SYMBOL
***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: t1 C: TYPE T: int D: 10
N: proc C: PROCEDURE T: short
N: MAIN C: PROCEDURE T: void
-->N: a C: VAR T: t1
-->N: c C: CONST T: short V: 11
